packages/x-components/src/x-modules/facets/store/types.ts

```diff
@@ -195,3 +207,20 @@ export interface MutateFilterPayload {
    */
   newFilterState: Partial<Filter>;
 }
+
+/**
+ * Configuration options for the {@link FacetsXModule}.
+ *
+ * @public
+ */
+export interface FacetsConfig {
+  /** The filter strategy to use when providing the selected filters for requests. */
+  filtersForRequestStrategy: FiltersForRequestStrategy;
```

Comment:
should be `filtersStrategyForRequest`? or `filtersStrategy`? or `filtersStrategyRequest`?

Reply:
I like `filtersStrategyForRequest` better, changing it 👍